.eslintrc.cjs

```diff
@@ -0,0 +1,57 @@
+module.exports = {
```

Comment:
Why does this have to be polluting the root of the repository?

We'll have a number of non-JS tools tools as well. Do we have to have this here?

Reply:
Will move to frontend apps workspace